src/hooks/registration.ts

```diff
@@ -48,11 +48,24 @@ export class ConvertNullToUndefinedAfterSuccessHook implements AfterSuccessHook
         return obj;
     }
 
+    private isResponseJson(response: Response): boolean {
+        const contentType = response.headers.get("content-type");
+        return !!(contentType && contentType.indexOf("application/json") !== -1);
```

Comment:
You could simplify this to something like:
```
return contentType?.includes("application/json") ?? false;
```

Reply:
done, thanks!